source/module_esolver/esolver_ks_lcao.h

```diff
@@ -109,6 +109,13 @@ namespace ModuleESolver
         // set matrix and grid integral
         void set_matrix_grid(Record_adj& ra);
 
+        void orb_paramter_init(double& dr_uniform, 
```

Comment:
make this an independent function, not a member function of esolver.

Reply:
The file location has been changed